src/returns/lines.js

```diff
@@ -65,7 +66,7 @@ const getMonths = (startDate, endDate) => {
     });
     datePtr.add(1, 'month');
   }
-  while (datePtr.isSameOrBefore(endDate, 'month'));
+  while (isPartialPeriod ? datePtr.isBefore(endDate, 'month') : datePtr.isSameOrBefore(endDate, 'month'));
```

Comment:
This line is quite long, I wonder if it would help readability to be something like:
```
const predicate = isPartialPeriod ? datePtr.isBefore : datePtr.isSameOrBefore;
while predicate(endDate, 'month');
```